frontend/src/pages/Search.js

```diff
@@ -73,8 +73,11 @@ export default function Search() {
     const accessChatHandler=(values)=>{
 
       const isPresent=state.find((data)=>{
+        console.log(data)
```

Comment:
Remove all console logs please !

Reply:
@balaraju-nerati Yeah remove all the console.logs present in frontend

Reply:
Hey, @SamarthKadam I've removed the console logs.

Reply:
LGTM